### ISSUE TITLE
Initialize project flow opens create_dbflow.sh script and does not initialize project structure under windows

### ISSUE THREAD

**Issue by yevon** [MaikMichel/dbFlux#14]
Hi! I was trying to setup the extension under windows 11. I have installed properly sqlcl and the bash command is recognized under the default WSL. The first time, there was some java asking for permissions to execute, and after that, a create_dbflow.sh is opened, but just it, the folder structure doesn't get created and nothing is imported. I can't see any details about what failed. What could it be? thanks!

**Comment by MaikMichel** [comment]
Sounds strange, maybe we should have a look at it via video call. To which email address can I send you a GoogleMeet invitation? And when is the best time for you (time zone)?

**Comment by yevon** [comment]
Yes sure! I'm in spain, GMT+2. I've added you in linkedin, I will provide my email to you there. This project looks amazing as it solves most of the problems I was looking to solve at my organization:

- PLSQL development in files under git repositories and vscode development
- Deployment between environments (Considering liquibase)
- Oracle apex version control and comparisons

I already tried Oradew extension but it did have some missing functionality that this project seems to solve. I would also want to get this functionalities, but current extensions don't seem to work properly:

- Fast code autocompletion within all plsql packages in the workspace (PLSQL language extension seems not updated and has bugs)
- Ctrl + click to open related functions or go to declaration.  (PLSQL language extension seems not updated and has bugs)
- SQL execution (The most performant seems SQL Tools)
- Native sqldeveloper for vscode continues to be unusably slow and buggy without proper files support.

**Comment by MaikMichel** [comment]
Hi [@brMichal](https://github.com/brMichael),

du hast doch auch Window 11. Kannst du ggf. Gabriel unterstützen?
